daemon/exec.go

```diff
@@ -45,29 +45,29 @@ func (d *Daemon) ExecExists(name string) (bool, error) {
 // with the exec instance is stopped or paused, it will return an error.
 func (d *Daemon) getExecConfig(name string) (*exec.Config, error) {
 	ec := d.execCommands.Get(name)
+	if ec != nil {
```

Comment:
I think that's supposed to be `if ec == nil`, isn't it? :smiley:

Reply:
I am wrong, 😄 
will update this soon.